core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -104,6 +105,7 @@
  * @since 4.0
  */
 @MessageLogger(projectCode = "ISPN")
+@ValidIdRange(min = 1, max = 1000)
```

Comment:
There is an overlap in range with infinispan-commons. They aren't duplicating the IDs, though.

Reply:
I've renumbered the offending logs in core and adjusted the ranges. 
I know, renumbering isn't ideal, but it's a handful of messages and it simplifies our life in the long run.